core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -33,15 +33,36 @@ oppia.animation('.oppia-collection-animate-slide', function() {
 oppia.controller('CollectionPlayer', [
   '$scope', '$http', 'ReadOnlyCollectionBackendApiService',
   'CollectionObjectFactory', 'CollectionPlaythroughObjectFactory',
-  'alertsService',
+  'alertsService', 'UrlInterpolationService',
   function($scope, $http, ReadOnlyCollectionBackendApiService,
     CollectionObjectFactory, CollectionPlaythroughObjectFactory,
-    alertsService) {
+    alertsService, UrlInterpolationService) {
     $scope.collection = null;
     $scope.collectionPlaythrough = null;
     $scope.collectionId = GLOBALS.collectionId;
     $scope.showingAllExplorations = !GLOBALS.isLoggedIn;
     $scope.previewCardIsShown = true;
+    $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
+    // The pathIconParameters is an array containing the co-ordinates,
+    // background color and icon url for the icons generated on the path
+    $scope.pathIconParameters = [];
+    $scope.activeHighlightedIconIndex = -1;
+    $scope.MIN_HEIGHT_FOR_PATH_SVG = 220;
```

Comment:
Add units -- MIN_HEIGHT_FOR_PATH_SVG_PX (assuming this is indeed pixels?); ditto below.